### DISCUSSION TITLE
JSON support

### DISCUSSION THREAD

**Discussion by bimgeek** [imartinez/privateGPT#805] [Q&A]
Hey everyone,

I have a json data that I want to do some queries. Does Private GPT support JSON?

**Comment by mzamini92** [comment]
Yes, PrivateGPT supports JSON. You can load a JSON file into PrivateGPT by using the load_json() function. This function will return a list of dictionaries, where each dictionary represents a single document in the JSON file.
```
import json
import privategpt

def main():
    with open('data.json', 'r') as f:
        data = json.load(f)

    documents = privategpt.load_json(data)

    for doc in documents:
        print(doc['name'])

if __name__ == '__main__':
    main()
```
